helmfile.d/jenkins-infra.yaml

```
@@ -8,14 +8,12 @@ releases:
     chart: jenkins/jenkins
     version: 3.8.8
     namespace: jenkins-infra
-    set:
-      - name: namespace
-        value: jenkins-infra
     values:
     - ../config/jenkins-infra.yaml
   - name: jenkins-infra-additional
     chart: jenkins-infra/jenkins-additional
     version: 0.0.1
     namespace: jenkins-infra
-    values:
-    - ../config/jenkins-infra-additional.yaml
+    set:
+      - name: clusterAdminEnabled
+        value: true
```

```suggestion
    values:
      - clusterAdminEnabled: true
```
I would have kept the separate values file, even if it contains only one value, in order to be coherent with the other helmfiles, but specify the only value like that is acceptable.